dali/pipeline/operators/reader/loader/sequence_loader.cc

```diff
@@ -89,18 +74,64 @@ std::vector<SequenceLoader::Stream> SequenceLoader::ParseStreams(string file_roo
   return streams;
 }
 
-std::vector<size_t> SequenceLoader::CalculateStreamSizes(const std::vector<Stream> &streams,
-                                                         size_t sample_lenght) {
-  std::vector<size_t> result;
+}  // namespace filesystem
+
+namespace detail {
+
```

Comment:
Maybe anonymous namespace for this? Seems to be widely used for `cc` files

Reply:
That way it won't be testable outside of that .cc file.